### PR TITLE
Prevent duplicate votes

### LINKED ISSUE #81: Bug in a user group adhesion
# This is a Bug Report 

#### :tophat: Description

When a User Group give support (adhere) to an initiative, the user_grpup apper twice in the adhesion list.

For bug reports:
* What went wrong?
* What did you expect should have happened?
Just appear one
* What was the config you used?
* What stacktrace or error message from your provider did you see?

#### :clipboard: Additional Data

* ***Decidim deployment where you found the issue***:
* ***Browser & version***:
* ***Screenshot***:
captura de pantalla de 2017-12-14 16-34-14

* ***Error messages***:
* ***URL to reproduce the error***: [https://meta.decidim.barcelona/initiatives/i-5?locale=ca](https://meta.decidim.barcelona/initiatives/i-5?locale=ca)

### PR DESCRIPTION
**Pull request by jsperezg**
#### :tophat: What? Why?

Votes table validates for unique records in rails, but not in db schema. There might happen that duplicated votes/supports are accepted.

This might happen when 
#### :pushpin: Related Issues
- Fixes #81

### CODE REVIEW

**Comment by mrcasals** [comment]
@jsperezg tests are failing, can you check them?

File: db/migrate/20171214161410_add_unique_on_votes.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        find_next(issue)&.destroy
+      end
+    end
+
+
+    add_index :decidim_initiatives_votes,
+              columns,
+              unique: true,
+              name: 'decidim_initiatives_voutes_author_uniqueness_index'
+  end
```
Comment (mrcasals) [inline db/migrate/20171214161410_add_unique_on_votes.rb]: Missing `down` action to remove the index

Comment (jsperezg) [reply db/migrate/20171214161410_add_unique_on_votes.rb]: THis index should never be removed. It is not a feature under test. It fixes a critical issue.

Comment (mrcasals) [reply db/migrate/20171214161410_add_unique_on_votes.rb]: Fair enough

**Pull request merged**